src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
according to BEM rules each BEM-component should be in a separate file (card, stars)